doc/source/user_guide/tutorials/animate/animate_data.rst

```diff
@@ -0,0 +1,503 @@
+--------------------
+
+Here we we will download a  result file available in our `Examples` package.
+For more information about how to import your result file in DPF check
+the :ref:`ref_tutorials_import_data` tutorial section.
+
+.. tab-set::
+
+    .. tab-item:: MAPDL
+
+
+        .. code-block:: python
+
+            # Import the ``ansys.dpf.core`` module, including examples files and the operators subpackage
+            from ansys.dpf import core as dpf
+            from ansys.dpf.core import examples
+            from ansys.dpf.core import operators as ops
+            # Define the result file
+            result_file_path_1 = examples.find_msup_transient()
+            # Create the model
+            my_model_1 = dpf.Model(data_sources=result_file_path_1)
+            # Get the mesh
+            my_meshed_region_1 = my_model_1.metadata.meshed_region
+
+    .. tab-item:: LSDYNA
+
+        .. code-block:: python
+
+            # Import the ``ansys.dpf.core`` module, including examples files and the operators subpackage
+            from ansys.dpf import core as dpf
+            from ansys.dpf.core import examples
+            from ansys.dpf.core import operators as ops
+            # Define the result file
+            result_file_path_2 = examples.download_d3plot_beam()
+            # Create the DataSources object
+            my_data_sources_2 = dpf.DataSources()
+            my_data_sources_2.set_result_file_path(filepath=result_file_path_2[0], key="d3plot")
+            my_data_sources_2.add_file_path(filepath=result_file_path_2[3], key="actunits")
+            # Create the model
+            my_model_2 = dpf.Model(data_sources=my_data_sources_2)
+
+Define time and mesh scopings
+-----------------------------
+
+Here we get all the the time steps of the |TimeFreqSupport| and all the |MeshedRegion| with results in a ``Nodal``
+location (only elemental, nodal or faces location are supported for the animation).
+For more information on how to define a scoping check the ``Narrow down data`` tutorial in the :ref:`ref_tutorials_import_data`
+tutorials section.
+
+.. tab-set::
+
+    .. tab-item:: MAPDL
+
+        .. code-block:: python
+
+            # Get all the time steps
+            time_scoping_1 = my_model_1.metadata.time_freq_support.time_frequencies
+            # Get all the mesh in a nodal location
+            mesh_scoping_1 = dpf.Scoping(ids=my_meshed_region_1.nodes.scoping.ids, location=dpf.locations.nodal)
+
+    .. tab-item:: LSDYNA
+
+        .. code-block:: python
+
+            # Get all the time steps
+            time_scoping_2 = my_model_2.metadata.time_freq_support.time_frequencies
+
+Extract the results
+-------------------
+
+You have to extracted the results of interest.
+
+When you animate the data you go through the |Fields| of a |FieldsContainer| and plot contours of
+the data norm or of the selected data component. This means that the geometry needs to be deformed
+based on the |Fields| themselves.
```

Comment:
We can animate without deformation. It will just animate the color map.

Reply:
Yes I explain it after. I can change the order of things here